samples/outgoing-webhook/nodejs/README.md

```diff
@@ -1,56 +1,43 @@
----
```

Comment:
Could you please revert this change? Seems like change from other sample.

Reply:
I see multiple files from samples/outgoing-webhook/nodejs/ folder. Please revert them all.